dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1603,3 +1603,18 @@ exception.occurred.while.running.ServletContainerInitializers.onStartup.useracti
 no.servlet.defined.for.servlet.filter.name.mapping=CWWWC0002W: No servlet definition is found for the {0} servlet name in the {1} filter mapping.
 no.servlet.defined.for.servlet.filter.name.mapping.explanation=The servlet name is used in filter mapping, but its definition has not been defined.
 no.servlet.defined.for.servlet.filter.name.mapping.useraction=Define the servlet which is used in the servlet name filter mapping.
+
+# {0} Cookie name that has conflict SameSite attribute
+# {1} current SameSite attribute value
+# {2} new SameSite attribute value
+duplicate.samesite.attribute.for.cookie.name=CWWWC0003W: A duplicate cookie name [{0}] with SameSite attribute value [{1}] is configured with new value [{2}].  No SameSite attribute will be added for this cookie.
+duplicate.samesite.attribute.for.cookie.name.explanation=A cookie name must not have multiple SameSite attribute values.  The attribute will not be added for the cookie.
```

Comment:
**change**
A cookie name must not have multiple SameSite attribute values.  The attribute will not be added for the cookie.
**to**
A cookie name cannot have multiple SameSite attribute values.  No attribute is added for the cookie.